infra/packer/holmusk-network.json

```
@@ -0,0 +1,58 @@
+{
+  "variables": {
+    "aws_access_key": "{{env `AWS_ACCESS_KEY_ID`}}",
+    "aws_secret_key": "{{env `AWS_SECRET_ACCESS_KEY`}}"
+  },
+  "builders": [{
+    "type": "amazon-ebs",
+    "access_key": "{{user `aws_access_key`}}",
+    "secret_key": "{{user `aws_secret_key`}}",
+    "region": "ap-southeast-1",
+    "source_ami": "ami-0efb24bbbf33a2fd7",
+    "instance_type": "m3.large",
+    "ssh_username": "ubuntu",
+    "ami_name": "holmusk-network-deploy-image",
+    "force_deregister": true,
+    "tags": {
+      "Name": "holmusk-network-deploy-image"
+    }
+  }],
+  "provisioners": [
+    {
+      "type": "shell",
+      "pause_before": 30,
+      "inline": [
+        "sudo apt-get update",
+        "export DEBIAN_FRONTEND=noninteractive && sudo -E apt-get -y -o Dpkg::Options::=\"--force-confdef\" -o Dpkg::Options::=\"--force-confold\" upgrade",
+        "sudo apt-get install nginx apt-transport-https gnupg-agent -y",
+        "curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -",
+        "sudo add-apt-repository -u \"deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs)  stable\"",
+        "curl -sL https://deb.nodesource.com/setup_12.x | sudo bash -",
+        "sudo apt-get install docker-ce docker-ce-cli containerd.io nodejs -y",
+        "mkdir -p /home/ubuntu/holmusk-network",
+        "mkdir -p /home/ubuntu/.aws",
+        "sudo mkdir -p /etc/datadog-agent/conf.d/journald.d",
+        "sudo rm -rf /etc/datadog-agent/datadog.yaml",
+        "echo \"169.254.169.254 instance-data\" | sudo tee -a /etc/hosts"
+      ]
+    },
+    {
+      "type": "file",
+      "source": "./",
+      "destination": "/home/ubuntu/holmusk-network/"
+    },
+    {
+      "type": "shell",
+      "inline": [
+        "cd /home/ubuntu/holmusk-network/backend/ && npm install",
+        "cd /home/ubuntu/holmusk-network/client/ && npm install",
+        "sudo cp /home/ubuntu/holmusk-network/infra/packer/holmusk-network-database.service /etc/systemd/system/holmusk-network-database.service",
+        "sudo cp /home/ubuntu/holmusk-network/infra/packer/holmusk-network-backend.service /etc/systemd/system/holmusk-network-backend.service",
+        "sudo systemctl daemon-reload",
+        "sudo systemctl enable holmusk-network-database.service",
+        "sudo systemctl enable holmusk-network-backend.service",
+        "sudo cp /home/ubuntu/holmusk-network/infra/packer/holmusk-network.conf /etc/nginx/sites-enabled/holmusk-network.conf"
```

```suggestion
        "sudo cp /home/ubuntu/holmusk-network/infra/packer/holmusk-network.conf /etc/nginx/sites-enabled/holmusk-network.conf",
        "sudo systemctl restart nginx"
```

We should restart nginx for it to pick up the new conf

---

There is no need to restart nginx as only an AMI but not operational EC2 instance is created.
Nginx will pick up new configuration when EC2 instance is deployed by Terraform.